diff -rupN Divi/et-pagebuilder/et-pagebuilder.php Divi-2.3.1-blog-module-post-order/et-pagebuilder/et-pagebuilder.php
--- Divi/et-pagebuilder/et-pagebuilder.php	2015-02-28 03:01:45.000000000 -0500
+++ Divi-2.3.1-blog-module-post-order/et-pagebuilder/et-pagebuilder.php	2015-02-28 03:44:29.000000000 -0500
@@ -6106,7 +6106,14 @@ echo <<<END
 				</div> <!-- .et-pb-option-container -->
 			</div> <!-- .et-pb-option -->
 
-			<!-- BEGIN CUSTOM MODIFICATION -->
+			<!-- 
+				BEGIN CUSTOM MODIFICATION
+				Blog Module Post Order Addon
+				Author: Dereck Martin <dmartin@burlyroot.com>
+				Website: http://www.burlyroot.com
+				Reference: https://www.elegantthemes.com/forum/viewtopic.php?f=25&t=377685
+			-->
+
 			<div class="et-pb-option">
 				<label for="et_pb_posts_order">
 END;
@@ -6119,7 +6126,7 @@ echo <<< END
 						<option value="ASC" <%= typeof( et_pb_posts_order ) !== 'undefined' && 'ASC' === et_pb_posts_order ? ' selected="selected"' : '' %>>Ascending</option>
 						<option value="DSC" <%= typeof( et_pb_posts_order ) !== 'undefined' && 'DESC' === et_pb_posts_order ? ' selected="selected"' : '' %>>Descending</option>
 					</select>
-				</div>
+
 				<p class="description">
 END;
 
diff -rupN Divi/functions.php Divi-2.3.1-blog-module-post-order/functions.php
--- Divi/functions.php	2015-02-28 03:08:16.000000000 -0500
+++ Divi-2.3.1-blog-module-post-order/functions.php	2015-02-28 03:44:30.000000000 -0500
@@ -3764,12 +3764,19 @@ function et_pb_blog( $atts ) {
 			'show_content' => 'off',
 			'show_author' => 'on',
 			'show_date' => 'on',
-			'posts_order' => 'ASC',
 			'show_categories' => 'on',
 			'show_pagination' => 'on',
 			'offset_number' => 0,
 			'background_layout' => 'light',
 			'show_more' => 'off',
+
+			/**
+			 * Blog Module Posts Order addon
+			 * Author: Dereck Martin <dmartin@burlyroot.com>
+			 * Website: http://www.burlyroot.com
+			 * Reference: https://www.elegantthemes.com/forum/viewtopic.php?f=25&t=377685
+			 */
+			'posts_order' => 'ASC',
 		), $atts
 	) );
 
